### DISCUSSION TITLE
请问如何调用类似showLoading这种方法？

### DISCUSSION THREAD

**Discussion by ybtsdst** [ecomfe/vue-echarts#821] [Q&A] [answered]
### 提前确认

- [X] 我可以确认这个新功能建议是提交给此 Vue 组件的，而不应该提交到 ECharts 项目本身。

### 新功能详情

vue小白；README中我看有一些方法（showLoading/hideLoading），请问如何调用？

**Comment by kingyue737** [comment, marked as answer]
通过[模板引用](https://cn.vuejs.org/guide/essentials/template-refs#ref-on-component)获得组件实例来调用组件实例上的方法。举个例子：
```vue
<script setup>
const chartRef = useTemplateRef('chart')

onMounted(() => {
  chartRef.value.showLoading()
})
</script>

<template>
  <VChart ref="chart" />
</template>
```
